SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -74,6 +75,16 @@ struct FSubobjectSchemaData
 	}
 };
 
+// Schema data related to a schema component
+USTRUCT()
```

Comment:
Why not just a regular struct?

Reply:
Or even just a map from id to array?

Reply:
Ah, this one is funny. Turns out you can't do that and then have it be a UPROPERTY. You have to wrap it in a USTRUCT, otherwise Unreal fails to parse this...

Reply:
Why do we need it to be a UProperty?

Reply:
To be saved in the schema database